coreneuron/nrnoc/multicore.cpp

```diff
@@ -459,27 +232,6 @@ void nrn_multithread_job(void* (*job)(NrnThread*)) {
     }
 #endif
 // clang-format on
-#else
-
-/* old implementation */
-#if USE_PTHREAD
-    if (nrn_thread_parallel_) {
-        nrn_inthread_ = 1;
-        for (i = 1; i < nrn_nthread; ++i) {
-            send_job_to_slave(i, job);
-        }
-        (*job)(nrn_threads);
-        WAIT();
-        nrn_inthread_ = 0;
-    } else { /* sequential */
-#else
-    {
-#endif
-        for (i = 1; i < nrn_nthread; ++i) {
-            (*job)(nrn_threads + i);
-        }
-        (*job)(nrn_threads);
-    }
```

Comment:
Could you verify if you haven't deleted too much here?

Reply:
I deleted too much indeed! :)